mongo/assets/configuration/spec.yaml

```diff
@@ -32,9 +32,11 @@ files:
       value:
         example:
         - <HOST>:<PORT>
-        type: array
-        items:
-          type: string
+        anyOf:
```

Comment:
This is the configuration spec. It is used to autogenerate the config example file and the config model code